test/checksum.cpp

```diff
@@ -1,7 +1,9 @@
 #include "checksum.h"
 #include <sys/types.h>
 #include <sys/stat.h>
+#if 0
```

Comment:
Ummm, this seems like this include should just be removed rather than doing a never include guard around it